shared/src/simulation/events/patient-transfer-to-hospital-successful.ts

```diff
@@ -0,0 +1,35 @@
+import { IsUUID } from 'class-validator';
+import { getCreate } from '../../models/utils';
+import { UUID, uuidValidationOptions } from '../../utils';
+import { IsLiteralUnion, IsValue } from '../../utils/validators';
+import {
+    AlivePatientTriageCategory,
+    alivePatientTriageCategoryAllowedValues,
+} from '../utils/alive-patient-triage-category';
+import type { SimulationEvent } from './simulation-event';
+
+export class PatientTransferToHospitalSuccessfulEvent
+    implements SimulationEvent
+{
+    @IsValue('patientTransferToHospitalSuccessfulEvent')
+    readonly type = 'patientTransferToHospitalSuccessfulEvent';
+
+    @IsLiteralUnion(alivePatientTriageCategoryAllowedValues)
+    readonly patientCategory: AlivePatientTriageCategory;
+
+    @IsUUID(4, uuidValidationOptions)
+    readonly originSimulatedRegion: UUID;
```

Comment:
Which region Id does this hold? `departure` or `management`?

Reply:
the region where the patient is from (i made the name more clear)